### ISSUE TITLE
fixup signup prompt for anonymous users

### ISSUE THREAD

**Issue by pdpinch** [mitodl/open-discussions#1222]
See [InVision](https://invis.io/9EN908ZQPAX) and [Zeplin](https://zpl.io/2ZzODOd)

### acceptance criteria:
- [ ] `Log In` text isn't displaying on login button
- [ ] Change "Become a Member" button to "Sign Up" to be consistent with other sign up UX
- [ ] Change prompt text to match Zeplin

**Comment by pdpinch** [comment]
This is blocked until the text is approved.

**Comment by mbertrand** [comment]
@pdpinch, `Sign up to join MIT Open now! Members can [upvote|comment|post|follow]. Signing up is free and easy` is really verbose for a small popup, doesn't fit as is on 1 line:

<img width="521" alt="screen shot 2018-09-20 at 9 48 40 am" src="https://user-images.githubusercontent.com/187676/45822902-7a80cb00-bcba-11e8-9b33-ab03768c3669.png">



I can split it into 3 lines but that results in a pretty tall popup:
<img width="316" alt="screen shot 2018-09-20 at 9 48 19 am" src="https://user-images.githubusercontent.com/187676/45822869-64730a80-bcba-11e8-8791-cc299d80cfec.png">

**Comment by pdpinch** [comment]
Sorry Matt, we’re still debating the text and the layout. I think you should pick up another issue until this is settled. 

If you already have a fix for the blank button, I suppose we could merge that. But it’s likely to get tweaked.

**Comment by pdpinch** [comment]
Ok, I think we have the final text for this:

<img width="612" alt="screen shot 2018-09-25 at 18 50 48" src="https://user-images.githubusercontent.com/430126/46034863-af808980-c0cf-11e8-84ec-44079d8c7488.png">

**Comment by pdpinch** [comment]
I'm unassigning Matt since he's out for a few days.

**Comment by pdpinch** [comment]
We have an updated design and text for this now, so I've unblocked it an updated the description.
